src/AbstractUntypedContainerImplementation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+namespace Laminas\ServiceManager;
+
+use Psr\Container\ContainerInterface;
+use function assert;
+
+/**
+ * @psalm-require-extends ServiceManager
+ */
+abstract class AbstractUntypedContainerImplementation implements ContainerInterface
```

Comment:
I wish I could make this internal but this would trigger psalm internal method calls all over upstream applications where `ServiceManager` and `PluginManager` injections are used...

Reply:
Can we at least document intended usage then? :D

Reply:
done.